packages/oui-tile/src/definition/tile-definition.component.js

```diff
@@ -0,0 +1,17 @@
+import controller from "./tile-definition.controller";
+import template from "./tile-definition.html";
+
+export default {
+    template,
+    controller,
+    bindings: {
+        term: "@?",
+        termPopover: "@?",
+        description: "@?",
+        actions: "=?"
+    },
+    transclude: {
+        descriptionSlot: "?dd",
```

Comment:
Maybe we should check with something else than `<dd>`.
What do you think about `<oui-tile-description>` ?